### PR TITLE
Update to rook v1.1.0

### LINKED ISSUE #30: Enable CSI storage
It's useful to have a CSI based storage enabled. CSI reached 1.0/GA in 1.13.

### PR DESCRIPTION
**Pull request by jascott1**
Adds handling for CSI version of Rook to create_stack.sh

Closes #30

Signed-off-by: Justin Scott <justin.a.scott@intel.com>

### CODE REVIEW

**Comment by mythi** [comment]
@jascott1 I don't have any good use/test case for this. The original reason for #30 was just to get cloud-native-setup to exercise also CSI. With that in mind, this PR is enough to resolve #30.

**Comment by jascott1** [comment]
Upgrading this to version v1.1.0 since CSI is now production ready and the default.

**Comment by jascott1** [comment]
This is ready for review.

File: clr-k8s-examples/create_stack.sh [hunk 1]
```diff
@@ -140,17 +152,13 @@ function storage() {
 		echo "Waiting for Rook OSD"
 		sleep 60
 	done
-	# set default storage class to rook-ceph-block
+
+	# set default storageclass
 	kubectl patch storageclass rook-ceph-block -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
-	create_pvc
-	# create and destroy pvc until successful
-	while [[ $(kubectl get pvc test-pv-claim --no-headers | grep Bound -c) -ne 1 ]]; do
```
Comment (NitinAtIntel) [inline clr-k8s-examples/create_stack.sh]: Good refactor...

**Comment by jascott1** [comment]
@krsna1729 Did you want to review?

**Pull request merged**